packages/storybook/src/components/card/card.css-template.ts

```diff
@@ -33,7 +34,7 @@ export const cssCard: ComponentImplementation<Card<TemplateResult>> = {
               </div>
             `}
           </div>
-          <div class="dso-card-content">${content}</div>
+          <div class="dso-rich-content">${content}</div>
```

Comment:
interessante change, even dubbelchecken of `.dso-card-content` niet toch gebruikt moet worden.

Reply:
`<div class="dso-card-content">` zit in de shadow. de `.dso-rich-content` is dus een optionele class. Er zit wel styling op in de css component `card.scss` om de margin-top eraf te halen.